reference/OpenAPI.json

```diff
@@ -31794,18 +31814,10 @@
           "metadata"
         ]
       },
-      "9_req_update_customers_bulk": {
-        "type": "array",
+      "customers_update_in_bulk_request_body": {
+        "type": "object",
         "title": "Update Customers in bulk Request Body",
         "description": "Request body schema for `/customers/bulk/async`.",
-        "items": {
```

Comment:
In my opinion, you have an error here. We should accept an array of customers like before. 

![image](https://github.com/voucherifyio/voucherify-openapi/assets/35666945/2111d461-8a97-48b8-8bd6-ea47ee10a94c)

Reply:
I think that for the main request object (the array) the name: `customers_update_in_bulk_request_body` will be suitable
But for the items / customers in the array the name is set to `customers_update_in_bulk_item_request_body` following the convention: `{resource}_{action}_{differentiator}_{request|response}_{body|param|query}`
it could be also `customers_update_in_bulk_customer_request_body` to be more precise.

Reply:
but currently the name is: customers_update_in_bulk_item_request_body